packages/babel-cli/src/babel/dir.js

```diff
@@ -125,10 +135,17 @@ export default function(commander, filenames, opts) {
     const filename = filenames[index];
 
     handle(filename, function(err) {
-      if (err) throw err;
+      if (err) throw err.toString();
```

Comment:
Since the stack trace of build errors isn't very helpful (lots of internal Babel stuff not necessarily helpful to the end user), I've decided to use only the stringified message. It works well on Node >=8. Node <=6 produces a totally unhelpful message anyway 🤷‍♂️

Reply:
Is it a common practice in other CLI tools? is this error here crashing the process directly & going right into stderr?

Reply:
Yup, crashes entire thing. Throwing `err.toString()` is definitely not something common, usually you end up creating your own error extending from `Error` and override stacktrace if necessary, for better error messages.

Reply:
so if this is direct exit point of the program & to avoid throwing non error it could be converted to smth like:
```js
console.error(err.toString())
process.exit(1)
```

although im not entirely convinced that this is a good change, logged value is still logged currently, right? Only with less-readable for most users stack trace. Personally I often take such stack traces and go into my node_modules to get better understanding of what happened if the error message is not descriptive enough (which is most of the times 😉)

Reply:
I just noticed we throw raw error without wrapping it with an Error object. I've pushed a change to adjust that

Here's how it looks now:
<img width="766" alt="screen shot 2018-02-27 at 23 41 15" src="https://user-images.githubusercontent.com/5106466/36759493-d976ae20-1c17-11e8-83b0-52a7a8e80b42.png">

imho better than:
<img width="840" alt="screen shot 2018-02-27 at 23 42 23" src="https://user-images.githubusercontent.com/5106466/36759513-e9c2f072-1c17-11e8-970c-20698c44ff7d.png">

Reply:
Definitely better!